pkg/cli/all_test.go

```diff
@@ -0,0 +1,127 @@
+	type fields struct {
+		Quiet  bool
+		Output string
+		All    bool
+	}
+	type args struct {
+		cmd    *cobra.Command
+		args   []string
+		client *testdata.MockClient
+	}
+	var _, w, _ = os.Pipe()
+	tests := []struct {
+		name           string
+		fields         fields
+		args           args
+		wantErr        bool
+		wantOut        string
+		commandContext client.CommandContext
+	}{
+		{
+			name: "acorn all", fields: fields{
+				All:    false,
+				Quiet:  false,
+				Output: "",
+			},
+			commandContext: client.CommandContext{
+				ClientFactory: &testdata.MockClientFactory{},
+				StdOut:        w,
+				StdErr:        w,
+				StdIn:         strings.NewReader("y\n"),
+			},
+			args: args{
+				args:   []string{},
+				client: &testdata.MockClient{},
+			},
+			wantErr: false,
+			wantOut: "\nAPPS:\nNAME      IMAGE     HEALTHY   UP-TO-DATE   CREATED    ENDPOINTS   MESSAGE\nfound                                      292y ago               \n\nCONTAINERS:\nNAME              APP       IMAGE     STATE     RESTARTCOUNT   CREATED    MESSAGE\nfound.container                                 0              292y ago   \n\nVOLUMES:\nNAME           APP-NAME   BOUND-VOLUME   CAPACITY   STATUS    ACCESS-MODES   CREATED\nfound.volume   found      found.volume   <nil>                               292y ago\n\nSECRETS:\nALIAS         NAME           TYPE      KEYS      CREATED\nfound.found   found.secret             []        292y ago\n",
+		},
+		{
+			name: "acorn all -i", fields: fields{
+				All:    false,
+				Quiet:  false,
+				Output: "",
+			},
+			commandContext: client.CommandContext{
+				ClientFactory: &testdata.MockClientFactory{},
+				StdOut:        w,
+				StdErr:        w,
+				StdIn:         strings.NewReader("y\n"),
+			},
+			args: args{
+				args:   []string{"-i"},
+				client: &testdata.MockClient{},
+			},
+			wantErr: false,
+			wantOut: "\nAPPS:\nNAME      IMAGE     HEALTHY   UP-TO-DATE   CREATED    ENDPOINTS   MESSAGE\nfound                                      292y ago               \n\nCONTAINERS:\nNAME              APP       IMAGE     STATE     RESTARTCOUNT   CREATED    MESSAGE\nfound.container                                 0              292y ago   \n\nVOLUMES:\nNAME           APP-NAME   BOUND-VOLUME   CAPACITY   STATUS    ACCESS-MODES   CREATED\nfound.volume   found      found.volume   <nil>                               292y ago\n\nSECRETS:\nALIAS         NAME           TYPE      KEYS      CREATED\nfound.found   found.secret             []        292y ago\n\nIMAGES:\nREPOSITORY   TAG       IMAGE-ID\ntestrepo     testtag   found-image1\n",
+		},
+		{
+			name: "acorn all -o yaml", fields: fields{
+				All:    false,
+				Quiet:  false,
+				Output: "",
+			},
+			commandContext: client.CommandContext{
+				ClientFactory: &testdata.MockClientFactory{},
+				StdOut:        w,
+				StdErr:        w,
+				StdIn:         strings.NewReader("y\n"),
+			},
+			args: args{
+				args:   []string{"-oyaml"},
```

Comment:
What's the reasoning behind not doing this?
```suggestion
				args:   []string{"-o", "yaml"},
```